README.md

```diff
@@ -3,6 +3,23 @@ The files in this repository provide support for syntax highlighting in Vim.
 They must be installed in your vim runtimepath to work.
 
 ## Instructions
+
+### Automated install
```

Comment:
I'd prefer to title this section "Using a Plugin Manager"

Reply:
Yeah, that is better.